src/create.ts

```diff
@@ -262,7 +267,15 @@ export default async function create(argv: yargs.Arguments<any>) {
   } else {
     await copyDir(NATIVE_FILES, folder);
 
-    if (type === 'cpp') {
+    if (type === 'js') {
+      await Promise.all(
+        [
+          `${folder}/android`,
+          `${folder}/ios`,
+          `${folder}/${options.project.podspec}.podspec`,
+        ].map(async (file) => await fs.remove(path.resolve(file)))
+      );
```

Comment:
Few things:
1. Use `path.join` instead of string interpolation for cross-platform compat
2. Extra async/await in the map is not needed
3. Extra `path.resolve` is not needed since it's already full path

```suggestion
      await Promise.all(
        [
          `android`,
          `ios`,
          `${options.project.podspec}.podspec`,
        ].map((file) => fs.remove(path.join(folder, file)))
      );
```